packages/fuel-gauge/src/contract-factory.test.ts

```diff
@@ -164,7 +164,20 @@ describe('Contract Factory', () => {
   });
 
   it('Creates a contract with initial storage (dynamic key)', async () => {
-    const factory = await createContractFactory();
+    using launched = await launchTestNode({
+      walletsConfig: {
+        amountPerCoin: 5_000_000,
+      },
+    });
```

Comment:
```suggestion
    using launched = await launchTestNode();
```

[Default](https://github.com/FuelLabs/fuels-ts/blob/7ae80b8a4af7a2b770bb687edf1d9db671cc4a60/packages/account/src/test-utils/setup-test-provider-and-wallets.ts#L33) `amountPerCoin` is `10_000_000_000` so you don't need to define this.

Reply:
755f8e0